x-pack/plugin/data-frame/src/main/java/org/elasticsearch/xpack/dataframe/transforms/DataFrameTransformTask.java

```diff
@@ -304,6 +303,7 @@ public synchronized void start(Long startingCheckpoint, boolean force, ActionLis
                 listener.onResponse(new StartDataFrameTransformTaskAction.Response(true));
             },
             exc -> {
+                logger.error("[" + transform.getId() + "] failed updating state to [" + state + "].", exc);
```

Comment:
Is it possible to use interpolation markers ("{}") here (and in other places) the same way as you did in line 355?

Reply:
Its not possible to use interpolation and have an exception too.

Reply:
A, ok. Understand.

Reply:
Use a `ParameterizedMessage` 

```suggestion
                logger.error(new ParameterizedMessage("[{}] failed updating state to [{}].", transform.getId(), state), exc);
```

Reply:
+1
I forgot about it. Seems like a perfect solution to maintain consistency of how the messages are crafted in this file.